app/src/main/res/values/strings.xml

```diff
@@ -10,6 +10,9 @@
     <string name="title_snap_to_route">Snap to route</string>
     <string name="description_snap_to_route">Use snap to route option to show smoother location updates.</string>
 
+    <string name="title_valhalla_mock_navigation">Mock Valhalla Navigation</string>
+    <string name="description_vallhalla_mock_navigation">Mock a Valhalla navigation session using a mock location engine.</string>
```

Comment:
I would suggest an other naming. Something like this:

Title: *Valhalla Navigation*
Description: *Using Valhalla engine server, to generate directions*

To mention the mock, is not relevant here, imo.

Reply:
Yes. Done!